src/NUnitFramework/Directory.Build.targets

```diff
@@ -0,0 +1,9 @@
+<Project>
+
+  <PropertyGroup>
+    <DefineConstants Condition="'$(TargetFramework)' != 'net35'">$(DefineConstants);TASK_PARALLEL_LIBRARY_API</DefineConstants>
+
+    <DefineConstants Condition="'$(TargetFrameworkIdentifier)' == '.NETFramework'">$(DefineConstants);THREAD_ABORT</DefineConstants>
```

Comment:
Question: This is a small difference between this here and it in Directory.Build.props in https://github.com/nunit/nunit/pull/3988/files, but I think I recall you saying in a different PR or issue that this is to fix an issue related to when `TargetFrameworkIdentifier` is initially defined, is that right?

Reply:
Done.